librz/config/config.c

```diff
@@ -386,6 +386,50 @@ static bool __is_true_or_false(const char *s) {
 	return s && (!rz_str_casecmp(s, "true") || !rz_str_casecmp(s, "false"));
 }
 
+RZ_API RzConfigNode *rz_config_set_b(RzConfig *cfg, const char *name, bool value) {
```

Comment:
It seems you don't allow insertion of new evals in this function, while it is allowed in rz_config_set_i.
I agree with this choice in general, however i'm not sure if only done here is the right thing. Maybe we should allow insertion of new vars also here and think about RzConfig later. I think also RzConfig would benefit from a more general refactoring similar to newshell :D

Reply:
I would kill insertion everywhere.

Reply:
I agree, but probably a separate PR and last time i tried it caused some problems here and there because in the current state it is not always clear where a eval is really created the first time.

Reply:
Ok, added insertion too.